src/writeData/subscriptions/components/JsonParsingForm.tsx

```diff
@@ -46,6 +46,17 @@ const JsonParsingForm: FC<Props> = ({formContent, updateForm, edit}) => {
   const numberType = 'Number'
   const dataTypeList = [stringType, numberType]
   const [dataTypeM, setDataTypeM] = useState(stringType)
+  const microsecondsType = 'MS'
```

Comment:
if you're looking for tighter TS support for these kinds of patterns, i think this works:
``` Javascript
enum Precision {
  Millisecond = 'MS'
  Second = 'S'
  Microsecond = 'US'
  Nanosecond = 'NS'
}

const precisionList = Object.keys(Precision)
```

if not, please disregard